### ISSUE TITLE
cpu useage 40%  high, but gpu  only 3%  , it seems not used the gpu ???

### ISSUE THREAD

**Issue by kaowob** [Plachtaa/seed-vc#182]
cpu useage 40%  high, but gpu  only 3%  , it seems not used the gpu ???

**Comment by TobiasPfeifer** [comment]
what script are you running?
Just add this snippet to your script (after the imports or run it in the python CLI):
```python
import torch
if not torch.cuda.is_available():
    raise Exception("No GPU available")
```

for train_v2 check your accelerator config
